cluster/images/etcd/Dockerfile

```diff
@@ -12,8 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM BASEIMAGE
+FROM BASEIMAGE as builder
+
+RUN apt-get update -y \
```

Comment:
For posterity, could we add a short comment explaining what we need bash for here? It wasn't immediately obvious to me.

Reply:
Done!. thanks.